docs-web/configuration/transportserver-resource.md

```diff
@@ -391,6 +391,27 @@ spec:
     port: 80
 ```
 
+Snippets can also be specified for a stream. In the example below, we use snippets to configure [access control](http://nginx.org/en/docs/stream/ngx_stream_access_module.html) and we set the [logging level](http://nginx.org/en/docs/http/ngx_http_limit_conn_module.html#limit_conn_zone) for cases when the server limits the number of connections in a TransportServer:
+
+
+```yaml
+apiVersion: k8s.nginx.org/v1alpha1
+kind: TransportServer
+metadata:
+  name: cafe
+spec:
+  host: cafe.example.com
+  streamSnippets: |
```

Comment:
could you also add an entry for `streamSnippets` to the table for the spec here https://docs.nginx.com/nginx-ingress-controller/configuration/transportserver-resource/#transportserver-specification ? `Sets a custom snippet in the stream context.` would do the job

Reply:
done